azure-pipelines.yml

```diff
@@ -25,6 +33,7 @@ variables:
   prefix: $[format('0.{0:yyyy}.{0:MMdd}', pipeline.startTime)]
   version: $[format('{0}.{1}', variables.prefix, counter(variables.prefix, 1))] # e.g. 0.2001.0203.4
   fileVersion: $[variables.version]
+  Codeql.Enabled: true
```

Comment:
Will codeql artifacts be uploaded for every pr build as well? If yes, Should we prevent that?

Reply:
it won't. looks like it gets uploaded only for default branch